### DISCUSSION TITLE
go-cqhttp支持获取到机器人QQ在其他客户端发送的信息吗?

### DISCUSSION THREAD

**Discussion by MangoAm** [Mrs4s/go-cqhttp#1903] [General]
1.类似mirai的同步消息链类型

(同步消息和普通消息一样, 但是由 Bot 账号的其他客户端发送的消息, 同步到 mirai 时产生的事件. 此类事发送人永远是 Bot 本身.)

2.或者获取全部客户端的聊天记录

mirai有这个功能,但是mirai是java版本,使用不如go方便.

**Comment by tom-snow** [comment]
你是指[这个吗 message: report-self-message: true](https://docs.go-cqhttp.org/guide/config.html#配置信息)

**Reply by MangoAm** [reply]
非常感谢, 好像就是这个功能, 我马上去测试
